pulp_container/app/registry_api.py

```diff
@@ -1008,18 +1008,24 @@ def put(self, request, path, pk=None):
             add_content_units = [str(manifest.pk)]
             remove_content_units = []
 
-        dispatched_task = dispatch(
+        immediate_task = dispatch(
```

Comment:
while "pending blobs" are not implemented yet, we could still take advantage of immediate tasks during blob upload and blob mount, couldn't we?

Reply:
I do expect the blobs uploads to happen i parallel, so they will probably collide too much on the repository resource. That is why i did not do it right away.
We could however allow them to be both immediate _and_ deferred...